playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -30,6 +33,7 @@
     command: oc scale --replicas=0 dc/docker-registry
 
   - name: Create S3 bucket
+    when: aws_create_bucket | bool
```

Comment:
Hi,
@dak1n1 For example on our project we only have a IAM with a restricted policy. We only have access to an existing bucket and only for few actions.
The S3_SECRET_ACCESS_KEY are required because we need it into config.yml generated from the j2 template.

So I think it is a good thing to have that (or ignore error for this task)